### ISSUE TITLE
Modifying manifests

### ISSUE THREAD

**Issue by cemo** [kz8s/tack#138]
I have an interesting case where my modified api servers manifests are reverted somehow. I wanted to enabled batch api for enabling cronjobs of kubernetes but had some issues.

1. What is the cause of this revert? 
2. What is the correct way to modify them?

**Comment by yagonobre** [comment]
1. Because cloud-init in coreos run every time that the machine complete the boot
2. Modify user-dat in etcd instances, you need to stop the instance, and create a new launch configuration to auto scaling.

**Comment by cemo** [comment]
I have a production setup. Can I do this operation in a safe manner? :)

**Comment by cemo** [comment]
```
terraform plan -target=module.etcd.aws_instance.etcd[0]
```

displays only 1 instance. Can I do in this way? What do you think?

**Comment by cemo** [comment]
https://github.com/kz8s/tack/issues/75#issuecomment-258558846

**Comment by yagonobre** [comment]
Yes, you can use this procedure. But the step one is outdated, tack no more storage manifests on s3, you will need to edit the user-data for worker nodes.

**Comment by yagonobre** [comment]
``` 
terraform plan -target=module.etcd.aws_instance.etcd[0]
```

I think that it cant work, because you need to stop the instance to modify user-data

**Comment by cemo** [comment]
when you change user data, terraform destroys and recreates instance. Do you think that it is doable in this way?

**Comment by yagonobre** [comment]
Yes, but you need to do it instance by instance in masters, or you will get etcd unhealthy

**Comment by cemo** [comment]
Actually that is why I wrote `etcd[0]` because I was considering to do one by one.

**Comment by yagonobre** [comment]
Nice, I think that it work

**Comment by cemo** [comment]
I will also upgrade cluster as well. The order is matters? I mean masters first than I should update workers?

**Comment by yagonobre** [comment]
the order is not matters
